.github/scripts/lint_cargo_toml.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env -S uv run -s
+
+# /// script
+# requires-python = ">=3.9"
+# dependencies = ["tomlkit==0.13.3"]
+# ///
+"""
+Usage:
+  ./sort_deps.py                # edits ./Cargo.toml in place
+  ./sort_deps.py path/to/Cargo.toml
+"""
+
+import re
+import sys
+from pathlib import Path
+
+from tomlkit import parse, dumps, key, item
+from tomlkit.items import Table, InlineTable
+
+def reorder_table_inplace(tbl: Table):
+    """Sort keys by (len(key), key) while preserving comments/formatting."""
```

Comment:
I think its easier to grok this as a reader if we sort by `key` (although it "looks clean" when by length I think that is less useful).

Reply:
I'd argue a single list (regardless of univariate/multivariate) that is alphabetical is most helpful.